README.md

```diff
@@ -206,4 +206,6 @@ The pipeline adds several labels:
 * You also need to provide the `DOCKER_IMAGE` secret specifying your Docker image name, e.g., `quay.io/[username]/nfs-subdir-external-provisioner`.
 
 
-
+## NFS limitations
+* The provisioned storage limit is not enforced. An application can expand to use all the available storage regardless of the provisioned size.
+* Storage resize/expansion operations are therefore also not supported. You will get an error: `Ignoring the PVC: didn't find a plugin capable of expanding the volume; waiting for an external controller to process this PVC.`
```

Comment:
i am wondering maybe we need to mentions those as `NFS Subdir External Provisioner limitations` or `Provisioner Limitations` in short, since those are not exactly nfs limitations per se. what do you think?

Reply:
They are based on the NFS limitations also, but fair point. I changed the topic to the shorter form and also added to word "pitfalls" to signal that this is something you should read and internalize as there are no training wheels.